package.json

```diff
@@ -20,7 +20,7 @@
         "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
         "format": "prettier --write \"**/*.+(js|jsx|ts|tsx|json|css|scss|md)\"",
         "chromatic": "chromatic --exit-zero-on-changes",
-        "prepare": "husky install && install-peers",
```

Comment:
we don't need to install peers, as i've moved those that are need to actual dependencies. Uninstalled install-peers and install-peers-cli